asn1/LedgerLedgerLotocol.asn

```diff
@@ -11,30 +11,40 @@ IMPORTS
     UInt256
     FROM GenericTypes
 
+    SideProtocolDate
+    FROM GenericPacket
+
+    InterledgerPacket
+    FROM GenericPacket
+
     Address,
     Timestamp
     FROM InterledgerTypes
 ;
 
-SideProtocolData ::= SEQUENCE OF SEQUENCE {
-  protocolName IA5String,
-  protocolData OCTET STRING
-}
-
-PacketRequest ::= SEQUENCE {
+FreeRequest ::= SEQUENCE {
+  fromId UInt128,
```

Comment:
Why are these `UInt128`s instead of ILP Addresses? Seems more confusing to have to do address mapping. Also, the address format is flexible enough that you could use shortened aliases if you wanted to.

See https://github.com/interledger/rfcs/pull/261